source/patterns/00-atoms/doi/doi.twig

```diff
@@ -0,0 +1,7 @@
+{%- import 'atoms-html' as html -%}
```

Comment:
The `doi` directory that contains this file needs to be moved into the `inline` directory.